letsencrypt/acme/messages2_test.py

```diff
@@ -75,6 +76,12 @@ def test_repr(self):
         self.assertEqual('MockConstant(a)', repr(self.const_a))
         self.assertEqual('MockConstant(b)', repr(self.const_b))
 
+    def test_equality(self):
```

Comment:
I will move self.const_a_prime down to the test_equality function using the self.MockConstant constructor.